src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimatorIO.h

```diff
@@ -64,11 +66,18 @@ struct Measurements
     Eigen::VectorXd encoders, encodersSpeed; /**< Joint position and joint velocity measurements */
     bool lfInContact{false}; /**< left foot contact state */
     bool rfInContact{false}; /**< right foot contact state */
-    
-    /** stamped contact status, 
-     * the usage of this map must be in a way 
-     * that every time an element is used, 
-     * it must be erased from the map 
+
+    /** stamped relative poses,
+     * the usage of this map must be in a way
+     * that every time an element is used,
+     * it must be erased from the map
```

Comment:
It is not clear to me whether the user has to do this or not. If so, please consider automatizing this in some way since it seems something easy to forget/miss.

Reply:
Actually, this is done at the implementation level, the user doesn't have to do it. I will update the docs.

Reply:
Updated the docs and explicitly called the clear() on these maps in advance() method in https://github.com/dic-iit/bipedal-locomotion-framework/pull/254/commits/10e3009d2a596e5fe5285718b91c776db4683086.